distributed/protocol/__init__.py

```diff
@@ -78,14 +78,3 @@ def _register_cupy():
 @cuda_deserialize.register_lazy("numba")
 def _register_numba():
     from . import numba
-
-
-@cuda_serialize.register_lazy("cudf")
-@cuda_deserialize.register_lazy("cudf")
-def _register_cudf():
-    import cudf
-
-    if LooseVersion(cudf.__version__) > "0.9":
-        from cudf.comm import serialize
```

Comment:
Don't we still want to import the serialization definitions in cudf?

Reply:
yes, i think we still need these lines

Reply:
Ah ok. Thought we might be doing this in cuDF too. Have added it back 🙂

Reply:
Actually is this not handled [here]( https://github.com/rapidsai/cudf/blob/1ead2d55562b9cd1b80568b86b8b51ab616f3cb8/python/cudf/cudf/comm/serialize.py#L10-L40 )?

Reply:
> Actually is this not handled here?

Right, but the `from cudf.comm import serialize` statement might be needed here to make sure the registration is triggered correctly.